src/blood-center/blood-center.controller.ts

```diff
@@ -78,9 +80,13 @@ export class BloodCenterController {
     description:
       'Creating new set of Blood Center Details failed. Please check request body',
   })
+  @ApiUnauthorizedResponse({
+    description: 'Requester is not authorized to save Blood Center Details',
+  })
   async createBloodDetailStatus(
     @Body() body: SaveBloodCenterDetailsDto,
+    @Headers('authorization') authHeader: string,
```

Comment:
Is it case-insensitive (i.e. will this accept `Authorization`)?

Reply:
Yes